demo/src/static/workers/flight-path-data-decoder.js

```diff
@@ -0,0 +1,55 @@
+importScripts('./util.js');
+const FLUSH_LIMIT = 20000;
+let result = [];
+let index = 0;
+let count = 0;
+
+onmessage = function(e) {
+  const lines = e.data.text.split('\n');
+
+  lines.forEach(function(line) {
+
+    if (!line) {
+      return;
+    }
+
+    var parts = line.split('\t');
+    var coords0 = parts[2].split('\x01').map(function(str) { return decodePolyline(str, 5) });
+    var coords1 = parts[3].split('\x01').map(function(str) { return decodePolyline(str, 1) });
+
+    coords0.forEach(function(lineStr, i) {
+      for (var j = 1; j < lineStr.length; j++) {
+        var prevPt0 = coords0[i][j - 1],
```

Comment:
Workers are not compiled, so we can't use es6 syntax. In fact I was thinking of replacing all `const` and `let` for browser compatibility. That is also why we did not set up linter for them. We could try bring them to es6 and compile each one. It feels that webpack is not designed for this.

Workers has always been the ugly side of the demos - the data is generated outside of this project, and their formats are simply hard coded in these workers.